### DISCUSSION TITLE
How to enable Machine Translation

### DISCUSSION THREAD

**Discussion by pippo73** [tolgee/tolgee-platform#1881] [Q&A] [answered]
Hello all, 
I've just installed the selfhosted version of your application, but I cannot understand how to enable the Machine Translation feature?
Could someone point me to the right direction? 
tnx all

**Comment by JanCizmar** [comment, marked as answer]
Hi! Thanks for the query! 🤘 

You have to provide the credentials for at least one of the providers. https://tolgee.io/platform/self_hosting/configuration#machine-translation
